changelog.md

```
@@ -1,5 +1,21 @@
+<!-- AUTO-GENERATED from changelog/*.yaml by ror-api — do not edit by hand; maintainers edit the per-version YAMLs in changelog/. -->
+
 # Changelog
 
+### (2026-06-12) What's new in **ROR 1.70.1**
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)